### PR TITLE
Adding tooltip to loc-components

### PR DESCRIPTION
**Pull request by seancrater**
image

Pretty much as simple as components get in here.

### CODE REVIEW

**Comment by seancrater** [comment]
@markuskobler Removed .DS_Store and added it to the .gitignore 👍 thank you!

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ public/*.css*
 public/*.html*
 
 coverage/
+.DS_Store
```
Comment (ntenczar) [inline .gitignore]: :+1:

**Comment by ntenczar** [comment]
nice work sean

**Comment by seancrater** [comment]
thanks @ntenczar

**Pull request merged**